packages/npm-dependencies-validation/src/utils/packageJsonUtil.ts

```diff
@@ -0,0 +1,116 @@
+import { access, readFile } from "fs/promises";
+import { constants } from "fs";
+import { join, dirname, sep, relative } from "path";
+import { VscodeFsUri, VscodeWsFolder } from "../types";
+
+const yarnManagerFiles = [
+  "yarn.lock",
+  ".yarnrc",
+  ".yarnrc.yml",
+  ".yarn",
+  "./node_modules/.yarn-integrity",
+];
+const pnpmManagerFiles = [
+  "pnpm-workspace.yaml",
+  "pnpm-lock.yaml",
+  ".pnpmfile.cjs",
+  "./node_modules/.pnpm",
+];
+const npm7Properties = ["workspaces"];
+
+async function readJsonFile(packageJsonUri: VscodeFsUri): Promise<any> {
+  try {
+    const packageJsonContent = await readFile(packageJsonUri.fsPath, "utf-8");
+    const content: { name: string } = JSON.parse(packageJsonContent);
+    return content;
+  } catch (error: any) {
+    return {};
+  }
+}
+
+async function isPathExist(packageJsonPath: string): Promise<boolean> {
+  try {
+    await access(packageJsonPath, constants.R_OK | constants.W_OK);
+    return true;
+  } catch (error) {
+    return false;
+  }
+}
+
+// async function findParentPackageJsonPath(
```

Comment:
remove commented out code, (it remains in history of PR if u will need it

Reply:
Done